### PR TITLE
fix(components): [cascader] fix large interval when multiple checkboxes

### LINKED ISSUE #9426: [Component] [cascader] 在el-form中，当el-cascader多选模式时，多选框与文本之间间距大
Bug Type: **`Component`**

## Environment

- Vue Version: `3.2.37`
- Element Plus Version: `2.2.12`
- Browser / OS: `chrome/104.0.5112.101`
- Build Tool: `Vue CLI`

### Related Component

- `el-cascader`

### Reproduction Link

[Element Plus Playground](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtZm9ybSA6bW9kZWw9XCJmb3JtXCIgbGFiZWwtd2lkdGg9XCIxMjBweFwiPlxuICAgIDxlbC1mb3JtLWl0ZW0+XG4gICAgXHQ8ZWwtY2FzY2FkZXIgOm9wdGlvbnM9XCJvcHRpb25zXCIgOnByb3BzPVwicHJvcHNcIiBjbGVhcmFibGU+XG4gICAgICAgIDx0ZW1wbGF0ZSAjZGVmYXVsdD1cInsgbm9kZSwgZGF0YSB9XCI+XG4gICAgICAgICAgPHNwYW4+e3sgZGF0YS5sYWJlbCB9fTwvc3Bhbj5cbiAgICAgICAgICA8c3BhbiB2LWlmPVwiIW5vZGUuaXNMZWFmXCI+ICh7eyBkYXRhLmNoaWxkcmVuLmxlbmd0aCB9fSkgPC9zcGFuPlxuICAgICAgICA8L3RlbXBsYXRlPlxuICAgICAgPC9lbC1jYXNjYWRlcj5cbiAgICA8L2VsLWZvcm0taXRlbT5cbiAgPC9lbC1mb3JtPlxuXG48L3RlbXBsYXRlPlxuXG48c2NyaXB0IGxhbmc9XCJ0c1wiIHNldHVwPlxuaW1wb3J0IHsgcmVhY3RpdmUgfSBmcm9tICd2dWUnXG4gIFxuY29uc3QgaXNMYWJlbGVkQnlGb3JtSXRlbSA9IGZhbHNlXG5cbi8vIGRvIG5vdCB1c2Ugc2FtZSBuYW1lIHdpdGggcmVmXG5jb25zdCBmb3JtID0gcmVhY3RpdmUoe1xuICBuYW1lOiAnJyxcbiAgcmVnaW9uOiAnJyxcbiAgZGF0ZTE6ICcnLFxuICBkYXRlMjogJycsXG4gIGRlbGl2ZXJ5OiBmYWxzZSxcbiAgdHlwZTogW10sXG4gIHJlc291cmNlOiAnJyxcbiAgZGVzYzogJycsXG59KVxuXG5jb25zdCBwcm9wcyA9IHsgbXVsdGlwbGU6IHRydWUgfVxuXG5jb25zdCBvcHRpb25zID0gW1xuICB7XG4gICAgdmFsdWU6IDEsXG4gICAgbGFiZWw6ICdBc2lhJyxcbiAgICBjaGlsZHJlbjogW1xuICAgICAge1xuICAgICAgICB2YWx1ZTogMixcbiAgICAgICAgbGFiZWw6ICdDaGluYScsXG4gICAgICAgIGNoaWxkcmVuOiBbXG4gICAgICAgICAgeyB2YWx1ZTogMywgbGFiZWw6ICdCZWlqaW5nJyB9LFxuICAgICAgICAgIHsgdmFsdWU6IDQsIGxhYmVsOiAnU2hhbmdoYWknIH0sXG4gICAgICAgICAgeyB2YWx1ZTogNSwgbGFiZWw6ICdIYW5nemhvdScgfSxcbiAgICAgICAgXSxcbiAgICAgIH0sXG4gICAgICB7XG4gICAgICAgIHZhbHVlOiA2LFxuICAgICAgICBsYWJlbDogJ0phcGFuJyxcbiAgICAgICAgY2hpbGRyZW46IFtcbiAgICAgICAgICB7IHZhbHVlOiA3LCBsYWJlbDogJ1Rva3lvJyB9LFxuICAgICAgICAgIHsgdmFsdWU6IDgsIGxhYmVsOiAnT3Nha2EnIH0sXG4gICAgICAgICAgeyB2YWx1ZTogOSwgbGFiZWw6ICdLeW90bycgfSxcbiAgICAgICAgXSxcbiAgICAgIH0sXG4gICAgICB7XG4gICAgICAgIHZhbHVlOiAxMCxcbiAgICAgICAgbGFiZWw6ICdLb3JlYScsXG4gICAgICAgIGNoaWxkcmVuOiBbXG4gICAgICAgICAgeyB2YWx1ZTogMTEsIGxhYmVsOiAnU2VvdWwnIH0sXG4gICAgICAgICAgeyB2YWx1ZTogMTIsIGxhYmVsOiAnQnVzYW4nIH0sXG4gICAgICAgICAgeyB2YWx1ZTogMTMsIGxhYmVsOiAnVGFlZ3UnIH0sXG4gICAgICAgIF0sXG4gICAgICB9LFxuICAgIF0sXG4gIH0sXG4gIHtcbiAgICB2YWx1ZTogMTQsXG4gICAgbGFiZWw6ICdFdXJvcGUnLFxuICAgIGNoaWxkcmVuOiBbXG4gICAgICB7XG4gICAgICAgIHZhbHVlOiAxNSxcbiAgICAgICAgbGFiZWw6ICdGcmFuY2UnLFxuICAgICAgICBjaGlsZHJlbjogW1xuICAgICAgICAgIHsgdmFsdWU6IDE2LCBsYWJlbDogJ1BhcmlzJyB9LFxuICAgICAgICAgIHsgdmFsdWU6IDE3LCBsYWJlbDogJ01hcnNlaWxsZScgfSxcbiAgICAgICAgICB7IHZhbHVlOiAxOCwgbGFiZWw6ICdMeW9uJyB9LFxuICAgICAgICBdLFxuICAgICAgfSxcbiAgICAgIHtcbiAgICAgICAgdmFsdWU6IDE5LFxuICAgICAgICBsYWJlbDogJ1VLJyxcbiAgICAgICAgY2hpbGRyZW46IFtcbiAgICAgICAgICB7IHZhbHVlOiAyMCwgbGFiZWw6ICdMb25kb24nIH0sXG4gICAgICAgICAgeyB2YWx1ZTogMjEsIGxhYmVsOiAnQmlybWluZ2hhbScgfSxcbiAgICAgICAgICB7IHZhbHVlOiAyMiwgbGFiZWw6ICdNYW5jaGVzdGVyJyB9LFxuICAgICAgICBdLFxuICAgICAgfSxcbiAgICBdLFxuICB9LFxuICB7XG4gICAgdmFsdWU6IDIzLFxuICAgIGxhYmVsOiAnTm9ydGggQW1lcmljYScsXG4gICAgY2hpbGRyZW46IFtcbiAgICAgIHtcbiAgICAgICAgdmFsdWU6IDI0LFxuICAgICAgICBsYWJlbDogJ1VTJyxcbiAgICAgICAgY2hpbGRyZW46IFtcbiAgICAgICAgICB7IHZhbHVlOiAyNSwgbGFiZWw6ICdOZXcgWW9yaycgfSxcbiAgICAgICAgICB7IHZhbHVlOiAyNiwgbGFiZWw6ICdMb3MgQW5nZWxlcycgfSxcbiAgICAgICAgICB7IHZhbHVlOiAyNywgbGFiZWw6ICdXYXNoaW5ndG9uJyB9LFxuICAgICAgICBdLFxuICAgICAgfSxcbiAgICAgIHtcbiAgICAgICAgdmFsdWU6IDI4LFxuICAgICAgICBsYWJlbDogJ0NhbmFkYScsXG4gICAgICAgIGNoaWxkcmVuOiBbXG4gICAgICAgICAgeyB2YWx1ZTogMjksIGxhYmVsOiAnVG9yb250bycgfSxcbiAgICAgICAgICB7IHZhbHVlOiAzMCwgbGFiZWw6ICdNb250cmVhbCcgfSxcbiAgICAgICAgICB7IHZhbHVlOiAzMSwgbGFiZWw6ICdPdHRhd2EnIH0sXG4gICAgICAgIF0sXG4gICAgICB9LFxuICAgIF0sXG4gIH0sXG5dXG5cbmNvbnN0IG9uU3VibWl0ID0gKCkgPT4ge1xuICBjb25zb2xlLmxvZygnc3VibWl0IScpXG59XG48L3NjcmlwdD5cbiIsImltcG9ydF9tYXAuanNvbiI6IntcbiAgXCJpbXBvcnRzXCI6IHt9XG59IiwiX28iOnt9fQ==)

### Steps to reproduce

在el-form中，el-cascader多选模式

### What is Expected?

多选框与文本之间间距正常，可参考不放在form中的时候间距

### What is actually happening?

多选框与文本之间间距大

### Additional comments

_(empty)_

### PR DESCRIPTION
**Pull request by Chuck-Lau**
closed #9426

Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by Tsong-LC** [comment]
[EX.](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtZm9ybSA6bW9kZWw9XCJmb3JtXCIgbGFiZWwtd2lkdGg9XCIxMjBweFwiPlxuICAgIDxlbC1mb3JtLWl0ZW0+XG4gICAgXHQ8ZWwtY2FzY2FkZXIgOm9wdGlvbnM9XCJvcHRpb25zXCIgOnByb3BzPVwicHJvcHNcIiBjbGVhcmFibGU+XG4gICAgICAgIDx0ZW1wbGF0ZSAjZGVmYXVsdD1cInsgbm9kZSwgZGF0YSB9XCI+XG4gICAgICAgICAgPHNwYW4+e3sgZGF0YS5sYWJlbCB9fTwvc3Bhbj5cbiAgICAgICAgICA8c3BhbiB2LWlmPVwiIW5vZGUuaXNMZWFmXCI+ICh7eyBkYXRhLmNoaWxkcmVuLmxlbmd0aCB9fSkgPC9zcGFuPlxuICAgICAgICA8L3RlbXBsYXRlPlxuICAgICAgPC9lbC1jYXNjYWRlcj5cbiAgICA8L2VsLWZvcm0taXRlbT5cbiAgPC9lbC1mb3JtPlxuICBcbiAgPGVsLWRpdmlkZXIgLz5cblxuICA8ZWwtY2FzY2FkZXIgOm9wdGlvbnM9XCJvcHRpb25zXCIgOnByb3BzPVwicHJvcHNcIiBjbGVhcmFibGU+XG4gICAgPHRlbXBsYXRlICNkZWZhdWx0PVwieyBub2RlLCBkYXRhIH1cIj5cbiAgICAgIDxzcGFuPnt7IGRhdGEubGFiZWwgfX08L3NwYW4+XG4gICAgICA8c3BhbiB2LWlmPVwiIW5vZGUuaXNMZWFmXCI+ICh7eyBkYXRhLmNoaWxkcmVuLmxlbmd0aCB9fSkgPC9zcGFuPlxuICAgIDwvdGVtcGxhdGU+XG4gIDwvZWwtY2FzY2FkZXI+XG5cbjwvdGVtcGxhdGU+XG5cbjxzY3JpcHQgbGFuZz1cInRzXCIgc2V0dXA+XG5pbXBvcnQgeyByZWFjdGl2ZSB9IGZyb20gJ3Z1ZSdcbiAgXG5jb25zdCBpc0xhYmVsZWRCeUZvcm1JdGVtID0gZmFsc2VcblxuLy8gZG8gbm90IHVzZSBzYW1lIG5hbWUgd2l0aCByZWZcbmNvbnN0IGZvcm0gPSByZWFjdGl2ZSh7XG4gIG5hbWU6ICcnLFxuICByZWdpb246ICcnLFxuICBkYXRlMTogJycsXG4gIGRhdGUyOiAnJyxcbiAgZGVsaXZlcnk6IGZhbHNlLFxuICB0eXBlOiBbXSxcbiAgcmVzb3VyY2U6ICcnLFxuICBkZXNjOiAnJyxcbn0pXG5cbmNvbnN0IHByb3BzID0geyBtdWx0aXBsZTogdHJ1ZSB9XG5cbmNvbnN0IG9wdGlvbnMgPSBbXG4gIHtcbiAgICB2YWx1ZTogMSxcbiAgICBsYWJlbDogJ0FzaWEnLFxuICAgIGNoaWxkcmVuOiBbXG4gICAgICB7XG4gICAgICAgIHZhbHVlOiAyLFxuICAgICAgICBsYWJlbDogJ0NoaW5hJyxcbiAgICAgICAgY2hpbGRyZW46IFtcbiAgICAgICAgICB7IHZhbHVlOiAzLCBsYWJlbDogJ0JlaWppbmcnIH0sXG4gICAgICAgICAgeyB2YWx1ZTogNCwgbGFiZWw6ICdTaGFuZ2hhaScgfSxcbiAgICAgICAgICB7IHZhbHVlOiA1LCBsYWJlbDogJ0hhbmd6aG91JyB9LFxuICAgICAgICBdLFxuICAgICAgfSxcbiAgICAgIHtcbiAgICAgICAgdmFsdWU6IDYsXG4gICAgICAgIGxhYmVsOiAnSmFwYW4nLFxuICAgICAgICBjaGlsZHJlbjogW1xuICAgICAgICAgIHsgdmFsdWU6IDcsIGxhYmVsOiAnVG9reW8nIH0sXG4gICAgICAgICAgeyB2YWx1ZTogOCwgbGFiZWw6ICdPc2FrYScgfSxcbiAgICAgICAgICB7IHZhbHVlOiA5LCBsYWJlbDogJ0t5b3RvJyB9LFxuICAgICAgICBdLFxuICAgICAgfSxcbiAgICAgIHtcbiAgICAgICAgdmFsdWU6IDEwLFxuICAgICAgICBsYWJlbDogJ0tvcmVhJyxcbiAgICAgICAgY2hpbGRyZW46IFtcbiAgICAgICAgICB7IHZhbHVlOiAxMSwgbGFiZWw6ICdTZW91bCcgfSxcbiAgICAgICAgICB7IHZhbHVlOiAxMiwgbGFiZWw6ICdCdXNhbicgfSxcbiAgICAgICAgICB7IHZhbHVlOiAxMywgbGFiZWw6ICdUYWVndScgfSxcbiAgICAgICAgXSxcbiAgICAgIH0sXG4gICAgXSxcbiAgfSxcbiAge1xuICAgIHZhbHVlOiAxNCxcbiAgICBsYWJlbDogJ0V1cm9wZScsXG4gICAgY2hpbGRyZW46IFtcbiAgICAgIHtcbiAgICAgICAgdmFsdWU6IDE1LFxuICAgICAgICBsYWJlbDogJ0ZyYW5jZScsXG4gICAgICAgIGNoaWxkcmVuOiBbXG4gICAgICAgICAgeyB2YWx1ZTogMTYsIGxhYmVsOiAnUGFyaXMnIH0sXG4gICAgICAgICAgeyB2YWx1ZTogMTcsIGxhYmVsOiAnTWFyc2VpbGxlJyB9LFxuICAgICAgICAgIHsgdmFsdWU6IDE4LCBsYWJlbDogJ0x5b24nIH0sXG4gICAgICAgIF0sXG4gICAgICB9LFxuICAgICAge1xuICAgICAgICB2YWx1ZTogMTksXG4gICAgICAgIGxhYmVsOiAnVUsnLFxuICAgICAgICBjaGlsZHJlbjogW1xuICAgICAgICAgIHsgdmFsdWU6IDIwLCBsYWJlbDogJ0xvbmRvbicgfSxcbiAgICAgICAgICB7IHZhbHVlOiAyMSwgbGFiZWw6ICdCaXJtaW5naGFtJyB9LFxuICAgICAgICAgIHsgdmFsdWU6IDIyLCBsYWJlbDogJ01hbmNoZXN0ZXInIH0sXG4gICAgICAgIF0sXG4gICAgICB9LFxuICAgIF0sXG4gIH0sXG4gIHtcbiAgICB2YWx1ZTogMjMsXG4gICAgbGFiZWw6ICdOb3J0aCBBbWVyaWNhJyxcbiAgICBjaGlsZHJlbjogW1xuICAgICAge1xuICAgICAgICB2YWx1ZTogMjQsXG4gICAgICAgIGxhYmVsOiAnVVMnLFxuICAgICAgICBjaGlsZHJlbjogW1xuICAgICAgICAgIHsgdmFsdWU6IDI1LCBsYWJlbDogJ05ldyBZb3JrJyB9LFxuICAgICAgICAgIHsgdmFsdWU6IDI2LCBsYWJlbDogJ0xvcyBBbmdlbGVzJyB9LFxuICAgICAgICAgIHsgdmFsdWU6IDI3LCBsYWJlbDogJ1dhc2hpbmd0b24nIH0sXG4gICAgICAgIF0sXG4gICAgICB9LFxuICAgICAge1xuICAgICAgICB2YWx1ZTogMjgsXG4gICAgICAgIGxhYmVsOiAnQ2FuYWRhJyxcbiAgICAgICAgY2hpbGRyZW46IFtcbiAgICAgICAgICB7IHZhbHVlOiAyOSwgbGFiZWw6ICdUb3JvbnRvJyB9LFxuICAgICAgICAgIHsgdmFsdWU6IDMwLCBsYWJlbDogJ01vbnRyZWFsJyB9LFxuICAgICAgICAgIHsgdmFsdWU6IDMxLCBsYWJlbDogJ090dGF3YScgfSxcbiAgICAgICAgXSxcbiAgICAgIH0sXG4gICAgXSxcbiAgfSxcbl1cblxuY29uc3Qgb25TdWJtaXQgPSAoKSA9PiB7XG4gIGNvbnNvbGUubG9nKCdzdWJtaXQhJylcbn1cbjwvc2NyaXB0PlxuIiwiUGxheWdyb3VuZE1haW4udnVlIjoiPHNjcmlwdCBzZXR1cD5cbmltcG9ydCBBcHAgZnJvbSAnLi9BcHAudnVlJ1xuaW1wb3J0IHsgc2V0dXBFbGVtZW50UGx1cyB9IGZyb20gJy4vZWxlbWVudC1wbHVzLmpzJ1xuc2V0dXBFbGVtZW50UGx1cygpXG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8QXBwIC8+XG48L3RlbXBsYXRlPlxuIiwiaW1wb3J0X21hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge1xuICAgIFwiZWxlbWVudC1wbHVzXCI6IFwiaHR0cHM6Ly9wcmV2aWV3LTk0MjgtZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5mdWxsLm1pbi5tanNcIixcbiAgICBcImVsZW1lbnQtcGx1cy9cIjogXCJ1bnN1cHBvcnRlZFwiXG4gIH1cbn0iLCJpbXBvcnQtbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7XG4gICAgXCJ2dWVcIjogXCJodHRwczovL2Nkbi5qc2RlbGl2ci5uZXQvbnBtL0B2dWUvcnVudGltZS1kb21AbGF0ZXN0L2Rpc3QvcnVudGltZS1kb20uZXNtLWJyb3dzZXIuanNcIixcbiAgICBcIkB2dWUvc2hhcmVkXCI6IFwiaHR0cHM6Ly9jZG4uanNkZWxpdnIubmV0L25wbS9AdnVlL3NoYXJlZEBsYXRlc3QvZGlzdC9zaGFyZWQuZXNtLWJ1bmRsZXIuanNcIixcbiAgICBcImVsZW1lbnQtcGx1c1wiOiBcImh0dHBzOi8vcHJldmlldy05NDI4LWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguZnVsbC5taW4ubWpzXCIsXG4gICAgXCJlbGVtZW50LXBsdXMvXCI6IFwidW5zdXBwb3J0ZWRcIixcbiAgICBcIkBlbGVtZW50LXBsdXMvaWNvbnMtdnVlXCI6IFwiaHR0cHM6Ly9jZG4uanNkZWxpdnIubmV0L25wbS9AZWxlbWVudC1wbHVzL2ljb25zLXZ1ZUAyL2Rpc3QvaW5kZXgubWluLmpzXCJcbiAgfSxcbiAgXCJzY29wZXNcIjoge31cbn0iLCJlbGVtZW50LXBsdXMuanMiOiJpbXBvcnQgeyBnZXRDdXJyZW50SW5zdGFuY2UgfSBmcm9tICd2dWUnXG5pbXBvcnQgRWxlbWVudFBsdXMgZnJvbSAnZWxlbWVudC1wbHVzJ1xuXG5sZXQgaW5zdGFsbGVkID0gZmFsc2VcbmF3YWl0IGxvYWRTdHlsZSgpXG5cbmV4cG9ydCBmdW5jdGlvbiBzZXR1cEVsZW1lbnRQbHVzKCkge1xuICBpZiAoaW5zdGFsbGVkKSByZXR1cm5cbiAgY29uc3QgaW5zdGFuY2UgPSBnZXRDdXJyZW50SW5zdGFuY2UoKVxuICBpbnN0YW5jZS5hcHBDb250ZXh0LmFwcC51c2UoRWxlbWVudFBsdXMpXG4gIGluc3RhbGxlZCA9IHRydWVcbn1cblxuZXhwb3J0IGZ1bmN0aW9uIGxvYWRTdHlsZSgpIHtcbiAgcmV0dXJuIG5ldyBQcm9taXNlKChyZXNvbHZlLCByZWplY3QpID0+IHtcbiAgICBjb25zdCBsaW5rID0gZG9jdW1lbnQuY3JlYXRlRWxlbWVudCgnbGluaycpXG4gICAgbGluay5yZWwgPSAnc3R5bGVzaGVldCdcbiAgICBsaW5rLmhyZWYgPSAnaHR0cHM6Ly9wcmV2aWV3LTk0MjgtZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5jc3MnXG4gICAgbGluay5hZGRFdmVudExpc3RlbmVyKCdsb2FkJywgcmVzb2x2ZSlcbiAgICBsaW5rLmFkZEV2ZW50TGlzdGVuZXIoJ2Vycm9yJywgcmVqZWN0KVxuICAgIGRvY3VtZW50LmJvZHkuYXBwZW5kKGxpbmspXG4gIH0pXG59IiwiX28iOnsic2hvd0hpZGRlbiI6dHJ1ZSwic3R5bGVTb3VyY2UiOiJodHRwczovL3ByZXZpZXctOTQyOC1lbGVtZW50LXBsdXMuc3VyZ2Uuc2gvYnVuZGxlL2luZGV4LmNzcyJ9fQ==)

**Comment by holazz** [comment]
Hi, I think it isn't a reasonable way to fix this bug, because this adds extra dom elements. Maybe it can be solved by adjusting the style. WDYT?

**Comment by Tsong-LC** [comment]
> Hi, I think it isn't a reasonable way to fix this bug, because this adds extra dom elements. Maybe it can be solved by adjusting the style. WDYT?

image
I think it should be consistent with the `radio` of the solution.it will be easy to understand.What do you think？

**Comment by holazz** [comment]
The empty `span` inside `radio` is to avoid rendering label, `checkbox` maybe no need.

**Comment by Tsong-LC** [comment]
The `Open Closed Principle` may need to be observed. I think.

**Comment by tolking** [comment]
This modification causes the label(`el-checkbox__label`) to have an extra 8px interval, maybe it should be changed in a different way

**Comment by Tsong-LC** [comment]
Sorry, I didn't think about it.

File: packages/theme-chalk/src/cascader-panel.scss [hunk 1]
```diff
@@ -124,7 +124,8 @@
     text-overflow: ellipsis;
   }
 
-  > .#{$namespace}-radio {
+  > .#{$namespace}-radio,
+  .#{$namespace}-checkbox {
```
Comment (holazz) [inline packages/theme-chalk/src/cascader-panel.scss:128]: I think you should separate style for checkbox and radio:

```scss
> .#{$namespace}-checkbox {
  margin-right: 0;
}

> .#{$namespace}-radio {
  margin-right: 0;

  .#{$namespace}-radio__label {
    padding-left: 0;
  }
}
```

Comment (Tsong-LC) [reply packages/theme-chalk/src/cascader-panel.scss]: Thanks for the review.😁

**Review by holazz — APPROVED** [review]

**Review by tolking — APPROVED** [review]

**Pull request merged**